pages/api/address/transactions/sync/[address].ts

```diff
@@ -35,7 +35,7 @@ export default async (req: NextApiRequest, res: NextApiResponse): Promise<void>
       }
       const addressString = parseAddress(req.query.address as string)
       void await upsertAddress(addressString)
-      const transactions = await syncTransactionsForAddress(addressString)
+      const transactions = await syncTransactionsAndPricesForAddress(addressString)
```

Comment:
It will, it won't be writing on the database as we pull from the blockchain, but it would take the same time as if we were writing on the database, just uses more memory the way it is.

Reply:
Why not do it incrementally as it was being done before? As I see this change only removes value. I understand that the first part is being done incrementally, but then the rest should be incremental too. Having to store +20k transactions in memory (and that is for one address. If we have multiple users with multiple such addresses, it would be much more) doesn't look like a good idea at all to me.

Reply:
changed to work as before, but better organized, no value removed now, explained in the last point of your revision below